stable/kubernetes-dashboard/templates/rolebinding.yaml

```diff
@@ -14,13 +14,17 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cluster-admin
+  name: {{ if .Values.rbac.clusterAdminRole -}}
+cluster-admin
```

Comment:
is this correct? it looks like broken yaml.

Reply:
Ues it is, the `-}}` construction says that all spaces after the line will be removed. And `{{-` – vice versa.

https://golang.org/pkg/text/template/#hdr-Text_and_spaces